tests/bsim/bluetooth/host/att/pipeline/dut/src/main.c

```diff
@@ -116,20 +114,20 @@ static void device_found(const bt_addr_le_t *addr, int8_t rssi, uint8_t type,
 	param = BT_LE_CONN_PARAM_DEFAULT;
 	err = bt_conn_le_create(addr, BT_CONN_LE_CREATE_CONN, param, &conn);
 	if (err) {
-		FAIL("Create conn failed (err %d)", err);
+		TEST_FAIL("Create conn failed (err %d)", err);
 		return;
 	}
 }
 
-static strucc bt_conn *connecc(void)
+static struct bt_conn *connecc(void)
 {
 	int err;
 	struct bt_conn *conn;
 
 	UNSET_FLAG(is_connected);
 
 	err = bt_le_adv_start(BT_LE_ADV_CONN_FAST_1, NULL, 0, NULL, 0);
-	ASSERT(!err, "Adving failed to start (err %d)\n", err);
+	TEST_ASSERT(!err, "Adving failed to start (err %d)\n", err);
```

Comment:
Does `TEST_ASSERT` require a newline, but `TEST_FAIL` doesn't?

Reply:
The newline should be removed, my script didn't remove it in some cases and manual inspection failed :)